.github/workflows/test_linux_job.yml

```diff
@@ -17,7 +17,8 @@ jobs:
       gpu-arch-type: cpu
       gpu-arch-version: ""
       script: |
-        export PATH="/opt/python/cp38-cp38/bin:${PATH}"
+        conda create -y -n test python=3.8
+        conda activate test
```

Comment:
The [test job](https://github.com/pytorch/test-infra/actions/runs/3236769746/jobs/5303019152#step:8:169) seems to have some issues:
```

Retrieving notices: ...working... done
+ conda activate test

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>
```

Sees like the shell is not properly configured to use conda?